ios/WebRTC.framework/Headers/RTCAudioSession.h

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
Instead of embedding the JitsiMeet and WebRTC frameworks, I suggest you use the JitsimeetSDK pod: https://cocoapods.org/pods/JitsiMeetSDK

Reply:
Hi @saghul, I tried to embedding frameworks because we go to remove buttons of toolbar :)